dev/integration_tests/ios_app_with_extensions/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -811,7 +946,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CURRENT_PROJECT_VERSION = "$(FLUTTER_BUILD_NUMBER)";
-				ENABLE_BITCODE = YES;
+				ENABLE_BITCODE = NO;
```

Comment:
Bitcode should be off for Runner.

Reply:
Why should bitcode be off? Bitcode is a requirement when you have a watch extension.

Reply:
Because this is an integration test that will never be submitted to the App Store and I didn't want to track down [issues related to the pods building before the correct bitcode-embedded version of Flutter gets copied around](https://github.com/flutter/flutter/issues/39356#issuecomment-618138976).  I'll set it back to `YES` and see if this particular integration test triggers that case.